[CI] ICUCollationKeywordFieldMapperIT.testNormalization() failure

Build logs: https://gradle-enterprise.elastic.co/s/tcbswba24vb6u/console-log#L5288

```
REPRODUCE WITH: ./gradlew ':plugins:analysis-icu:integTestRunner' --tests "org.elasticsearch.index.mapper.ICUCollationKeywordFieldMapperIT.testNormalization" -Dtests.seed=534D27A7291B436E -Dtests.security.manager=true -Dtests.locale=he -Dtests.timezone=Mexico/BajaNorte -Dcompiler.java=12
--
2> java.lang.AssertionError: Expected id: 2 at position 0 but wasn't. Total shards: 4 Successful shards: 4 & 0 shard failures:
Expected: "2"
but: was "1"
at __randomizedtesting.SeedInfo.seed([534D27A7291B436E:77124CFBC84F3C44]:0)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertOrderedSearchHits(ElasticsearchAssertions.java:247)
at org.elasticsearch.index.mapper.ICUCollationKeywordFieldMapperIT.testNormalization(ICUCollationKeywordFieldMapperIT.java:208)
```

This test has failed 6 times now today according to [build stats](https://build-stats.elastic.co/app/kibana#/discover/c46fbf60-05ec-11ea-ba75-91621e16ca05?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-30d,mode:quick,to:now))&_a=(columns:!(_source),filters:!(),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'project:elasticsearch%20AND%20class:*ICUCollationKeywordFieldMapperIT%20AND%20test:testNormalization'),sort:!(time,desc))).

--- a/plugins/analysis-icu/src/test/java/org/elasticsearch/index/mapper/ICUCollationKeywordFieldMapperIT.java
+++ b/plugins/analysis-icu/src/test/java/org/elasticsearch/index/mapper/ICUCollationKeywordFieldMapperIT.java
@@ -189,7 +189,7 @@ public class ICUCollationKeywordFieldMapperIT extends ESIntegTestCase {
             client().prepareIndex(index).setId("1")
                 .setSource("{\"id\":\"1\",\"collate\":\"" + equivalent[0] + "\"}", XContentType.JSON),
             client().prepareIndex(index).setId("2")
-                .setSource("{\"id\":\"1\",\"collate\":\"" + equivalent[1] + "\"}", XContentType.JSON)
+                .setSource("{\"id\":\"2\",\"collate\":\"" + equivalent[1] + "\"}", XContentType.JSON)
         );
 
         // searching for either of the terms should return both results since they collate to the same value

